tests/tests/test_utils.py

```diff
@@ -96,16 +99,56 @@ def test_serialize_exceptions(self) -> None:
             SyntaxError("Wrong"),
             ImproperlyConfigured("It's wrong"),
             InvalidTaskError(""),
-            SystemExit(),
         ]:
             with self.subTest(exc):
                 data = utils.exception_to_dict(exc)
                 self.assertEqual(utils.json_normalize(data), data)
-                self.assertEqual(set(data.keys()), {"exc_type", "exc_args"})
+                self.assertEqual(
+                    set(data.keys()), {"exc_type", "exc_args", "exc_traceback"}
+                )
                 reconstructed = utils.exception_from_dict(data)
                 self.assertIsInstance(reconstructed, type(exc))
                 self.assertEqual(reconstructed.args, exc.args)
 
+                # Check that the exception traceback contains only one line,
+                # the one with the name of the exception and the message.
+                dotted_path = utils.get_module_path(exc.__class__).replace(
+                    "builtins.", ""
+                )
+                msg = f": {exc.args[0]}\n" if exc.args[0] else "\n"
+                self.assertEqual(data["exc_traceback"], [dotted_path + msg])
+
+    def test_serialize_full_traceback(self) -> None:
+        try:
+            # Using optparse to generate an error because:
+            # - it's pure python
+            # - it's easy to trip down
+            # - it's unlikely to change ever
+            # - We can normalize the strack trace from it
+            optparse.OptionParser(option_list=[1])  # type: ignore
+        except Exception as e:
+            traceback = utils.exception_to_dict(e)["exc_traceback"]
+
+            expected_traceback_head = "Traceback (most recent call last):\n"
```

Comment:
Nit: Defining these variables separately is unnecessary - just define them in the assertions.

Alternatively, why split them out at all? Just have 1 assertion.

Reply:
Because those parts of the traceback are constant, while there are a few lines that will depend on the setup. I'll add a comment about this.